### PR TITLE
chore: Spelling

### PR DESCRIPTION
**Pull request by jsoref**
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

I can rewrite commit messages to match house style once changes are considered.

#### What does this PR do?

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/snyk-cli/commit/66ebc350a21d2a726a31b4d605fc8491aaaa55c4#commitcomment-87618095

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/snyk-cli/commit/c24796eb578ce743b2b3227c8a2349938b49c7e5
 
Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

#### Where should the reviewer start?

Identify any files that shouldn't be changed.

### CODE REVIEW

File: test/acceptance/workspaces/fail-on/docker/no-fixable/vulns-result.json [hunk 1]
```diff
@@ -1660,7 +1660,7 @@
...
+        "description": "## Overview\nGNU Libc current is affected by: Re-mapping current loaded library with malicious ELF file. The impact is: In worst case attacker may evaluate privileges. The component is: libld. The attack vector is: Attacker sends 2 ELF files to victim and asks to run ldd on it. ldd execute code.\n\n## References\n- [CONFIRM](https://support.f5.com/csp/article/K11932200?utm_source=f5support&amp%3Butm_medium=RSS)\n- [Debian Security Tracker](https://security-tracker.debian.org/tracker/CVE-2019-1010023)\n- [MISC](https://sourceware.org/bugzilla/show_bug.cgi?id=22851)\n- [Security Focus](http://www.securityfocus.com/bid/109167)\n",
```
Comment (jsoref) [inline test/acceptance/workspaces/fail-on/docker/no-fixable/vulns-result.json:1663]: It's possible that files in this path should be skipped.

File: test/acceptance/workspaces/fail-on/pinnable/vulns-result.json [hunk 2]
```diff
@@ -767,7 +767,7 @@
...
...wntime.\r\n\r\nOne popular Denial of Service vulnerability is DDoS (a Distributed Denial of Service), an attack that attempts to clog network pipes to the system by generating a large volume of traffic from many machines.\r\n\r\nWhen it comes to open source libraries, DoS vulnerabilities allow attackers to trigger such a crash or crippling of the service by using a flaw either in the application code or from the use of open source libraries.\r\n\r\nTwo common types of DoS vulnerabilities:\r\n\r\n* High CPU/Memory Consumption- An attacker sending crafted requests that could cause the system to take a disproportionate amount of time to process. For example, [commons-fileupload:commons-fileupload](SNYK-JAVA-COMMONSFILEUPLOAD-30082).\r\n\r\n* Crash - An attacker sending crafted requests that could cause the system to crash. For Example,  [npm `ws` package](npm:ws:20171108)\n\n## References\r\n- [Django Vulnerability Description](https://www.djangoproject.com/weblog/2015/mar/18/security-releases/)\r\n- [Oracle Security Bulletin](http://www.oracle.com/technetwork/topics/security/bulletinapr2015-2511959.html)\r\n- [CVE](https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2015-2316)\r\n",
```
Comment (jsoref) [inline test/acceptance/workspaces/fail-on/pinnable/vulns-result.json:770]: It's possible that /vulns.*json should be skipped...

File: test/acceptance/workspaces/ruby-app-thresholds/test-result-low-severity.json [hunk 3]
```diff
@@ -298,7 +298,7 @@
       "publicationTime": "2017-09-14T09:12:31.394000Z",
       "references": [
         {
-          "title": "Github Commit",
+          "title": "GitHub Commit",
```
Comment (jsoref) [inline test/acceptance/workspaces/ruby-app-thresholds/test-result-low-severity.json:301]: Brand...

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

File: help/cli-commands/iac.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 ## Description
 
-The `snyk iac` commands find and report security issues in Infrastructure as Code files; detect, track, and alert on infrastructure drift and unmanaged resources; and create a .driftigore file.
+The `snyk iac` commands find and report security issues in Infrastructure as Code files; detect, track, and alert on infrastructure drift and unmanaged resources; and create a .driftignore file.
```
Comment (teodora-sandu) [inline help/cli-commands/iac.md:9]: I could be wrong but I think the help markdown is generated from GitBook, so this might need to change in GitBook

**Review by teodora-sandu — APPROVED** [review]

**Review by gablaxian — APPROVED** [review]

**Comment by thisislawatts** [comment]
🧹 Without any activity in more than 12 months, closing as stale.

**Comment by jsoref** [comment]
Sorry. I'm happy to update if there's a willingness to review...

**Pull request closed without merging**